bin/configs/java-native-8.yaml

```diff
@@ -0,0 +1,12 @@
+generatorName: java
+outputDir: samples/openapi3/client/petstore/java/native-java8
+library: native
+inputSpec: modules/openapi-generator/src/test/resources/3_0/java/petstore-with-fake-endpoints-models-for-testing-with-http-signature.yaml
+templateDir: modules/openapi-generator/src/main/resources/Java
+additionalProperties:
+  artifactId: petstore-openapi3-native-java8
+  hideGenerationTimestamp: true
+  serverPort: "8082"
+  dateLibrary: java8
```

Comment:
@bgong-mdsol I think you can remove this option as it makes no difference in the `native` library

Reply:
updated as suggested in https://github.com/OpenAPITools/openapi-generator/pull/7263/commits/0ddb2a9754c3085dd0f14da7a9b42982965620fb

Reply:
> Can you please resolve the merge conflicts? I'll try to review and merge if no question from me later this week.

@wing328 Sure, I am going to resolve the merge conflicts.